src/cli/cli.cpp

```diff
@@ -6216,15 +6247,37 @@ template <> otError Interpreter::Process<Cmd("region")>(Arg aArgs[])
 exit:
     return error;
 }
+    /**
+     * @cli releaserouterid (routerid)
+     * @code
+     * releaserouterid 16
+     * Done
+     * @endcode
+     * @cparam releaserouterid [@ca{routerid}]
+     * @par api_copy
+     * #otThreadReleaseRouterId
+     */
+{
```

Comment:
Looks like this bracket got moved and is causing all the failures.  It should be on line 6264 (between template and return).